classifier_and_summarizer/summarizer/summarizer_output.py

```diff
@@ -46,22 +47,26 @@ class StampPage:
     * stamp_position: The order or index of the stamp in the final
     collected of stamp pages
     '''
+
     def __init__(
             self,
             media_index,
+            sentence_index,
```

Comment:
Add a note on this in the class's docstring.

Reply:
Updated accordingly.